components/homme/src/share/compose_mod.F90

```diff
@@ -341,6 +341,9 @@ subroutine compose_query_bufsz(sendsz, recvsz)
     call cedr_query_bufsz(ssz, rsz)
     sendsz = max(sendsz, ssz)
     recvsz = max(recvsz, rsz)
+#else
+    sendsz = 0
+    recvsz = 0
```

Comment:
```
/qfs/people/mame117/repos/E3SM-tests/components/homme/src/share/compose_mod.F90(334): warning #6843: A dummy argument with an explicit INTENT(OUT) declaration is not given an explicit value.   [SENDSZ]
  subroutine compose_query_bufsz(sendsz, recvsz)
---------------------------------^
/qfs/people/mame117/repos/E3SM-tests/components/homme/src/share/compose_mod.F90(334): warning #6843: A dummy argument with an explicit INTENT(OUT) declaration is not given an explicit value.   [RECVSZ]
  subroutine compose_query_bufsz(sendsz, recvsz)
-----------------------------------------^
```

Reply:
I haven't looked at the code to know how these variables are being used. It is probably okay but @ambrad or @oksanaguba might know if it is okay to assign these to zero.

Reply:
Yes, these lines are fine. The subroutine isn't even called in this branch, but it's good to take care of compiler warnings. Thanks! (And thanks for tagging me @singhbalwinder.)